### PR TITLE
Mini-Rewrite of Skinning Tutorial

### PR DESCRIPTION
**Pull request by Circleen**

### CODE REVIEW

File: wiki/Skinning_Tutorial/en.md [hunk 1]
```diff
@@ -6,16 +6,16 @@
...
 
-Contrary to popular belief, to get a better understanding of how skinning works, you should start by mixing skins (copying other skin files and paste them into your newly created skin folder). **It must be noted that you are not allowed to share these in the forums.** However, mixing skins for yourself can help you learn what skinning can and can not achieve in osu!.
+Contrary to popular belief, to get a better understanding of how skinning works, you should start by mixing skins (copying other skin assets and pasting them into your skin folder). **You are `not` allowed to share these in the forums.** However, mixing skins for yourself can help you learn what skinning can and can not achieve in osu!.
 
-## Making a skin
+## Making A Skin
```
Comment (cl8n) [inline wiki/Skinning_Tutorial/en.md]:
```suggestion
## Making a skin
```
headings should be sentence cased

File: wiki/Skinning_Tutorial/en.md [hunk 2]
```diff
@@ -6,16 +6,16 @@
 
 ## Getting Started
```
Comment (cl8n) [inline wiki/Skinning_Tutorial/en.md]:
```suggestion
## Getting started
```

File: wiki/Skinning_Tutorial/en.md [hunk 3]
```diff
@@ -6,16 +6,16 @@
...
 
-Contrary to popular belief, to get a better understanding of how skinning works, you should start by mixing skins (copying other skin files and paste them into your newly created skin folder). **It must be noted that you are not allowed to share these in the forums.** However, mixing skins for yourself can help you learn what skinning can and can not achieve in osu!.
+Contrary to popular belief, to get a better understanding of how skinning works, you should start by mixing skins (copying other skin assets and pasting them into your skin folder). **You are `not` allowed to share these in the forums.** However, mixing skins for yourself can help you learn what skinning can and can not achieve in osu!.
```
Comment (cl8n) [inline wiki/Skinning_Tutorial/en.md]:
```suggestion
Contrary to popular belief, to get a better understanding of how skinning works, you should start by mixing skins (copying other skin assets and pasting them into your skin folder). **You are not allowed to share these in the forums.** However, mixing skins for yourself can help you learn what skinning can and can not achieve in osu!.
```
code blocks aren't for emphasis (and in this case I don't think any is needed)

File: wiki/Skinning_Tutorial/en.md [hunk 4]
```diff
@@ -24,13 +24,13 @@ Once you have your own skinning folder in the `Skins/` folder, you will first ne
 
 ---
 
-The next thing you will need is a theme or an idea. You should have one in mind to create an objective for what you want your skin to look like. This is also to help yourself when you need to gather and/or create assets.
+The most important thing you will need is a theme or an idea. You should have one in mind to create an objective for what you want your skin to look like. This is also to help yourself when you need to gather and/or create assets.
 
-The last thing to have ready is patience and a lot of time. New skins do not complete in a snap but can take a week or even several months, so give it some time.
+The last thing to have ready is patience and lots of time. New skins do not complete in a snap but can take a week or even several months, so give it some time.
```
Comment (cl8n) [inline wiki/Skinning_Tutorial/en.md]: not sure what the point is

File: wiki/Skinning_Tutorial/en.md [hunk 5]
```diff
@@ -24,13 +24,13 @@ Once you have your own skinning folder in the `Skins/` folder, you will first ne
...
 
 ## What's next?
 
-From here, you can either checkout the [skinning elements lists](/wiki/skinning#skin-elements-lists) to get an idea for what to include in your skin or go through the some of the tutorials:
+From here, you can checkout the [Skinning Elements Lists](/wiki/skinning#skin-elements-lists) to get an idea for what to include in your skin or go through the some of the tutorials:
```
Comment (cl8n) [inline wiki/Skinning_Tutorial/en.md]:
```suggestion
From here, you can check out the [skin elements lists](/wiki/Skinning#skin-elements-lists) to get an idea for what to include in your skin, or go through the some of the tutorials:
```

File: wiki/Skinning_Tutorial/en.md [hunk 6]
```diff
@@ -6,16 +6,16 @@
 
 ## Getting Started
 
-To get started, open the game client, select [Options](/wiki/Options), and click `Open osu! folder`. Once the `osu!/` folder is opened, go into the `Skins/` folder and create a new folder. You can name it whatever you want, but keep in mind that osu! will display this name in the options. Inside this folder, you can add any skinning elements listed in the skin element lists.
+To get started, open File Explorer. enter %localappdata% on the path bar, and press enter. Then, go into the `osu!/` folder, then `Skins/`. Right click, and create a new folder. You can also go into osu!, settings, and open the skins folder, then make the folder. You can name it whatever you want, but keep in mind that osu! will display this name on the dropdown of the skin select. Inside this folder, you can add any of the skinning elements listed in the [Skinning Elements Lists](/wiki/Skinning#skin-elements-lists).
```
Comment (cl8n) [inline wiki/Skinning_Tutorial/en.md]: shouldn't assume where the game is installed, the gist of the old version is fine for finding the skins folder (language is oddly casual, though)

File: wiki/Skinning_Tutorial/en.md [hunk 7]
```diff
@@ -24,13 +24,13 @@ Once you have your own skinning folder in the `Skins/` folder, you will first ne
 
 ---
 
-The next thing you will need is a theme or an idea. You should have one in mind to create an objective for what you want your skin to look like. This is also to help yourself when you need to gather and/or create assets.
+The most important thing you will need is a theme or an idea. You should have one in mind to create an objective for what you want your skin to look like. This is also to help yourself when you need to gather and/or create assets.
```
Comment (sr229) [inline wiki/Skinning_Tutorial/en.md]: Theme is rather not so important as skins are mostly aimed for tailoring the user experience for the author.

File: wiki/Skinning_Tutorial/en.md [hunk 8]
```diff
@@ -4,18 +4,18 @@
...
 
-Contrary to popular belief, to get a better understanding of how skinning works, you should start by mixing skins (copying other skin files and paste them into your newly created skin folder). **It must be noted that you are not allowed to share these in the forums.** However, mixing skins for yourself can help you learn what skinning can and can not achieve in osu!.
+Contrary to popular belief, to get a better understanding of how skinning works, you should start by mixing skins (copying other skin assets and pasting them into your skin folder). **You are not allowed to share these in the forums.** However, mixing skins for yourself can help you learn what skinning can and can not achieve in osu!.
```
Comment (sr229) [inline wiki/Skinning_Tutorial/en.md]: Bolded reminder is too imposing. Recommended you rewrite this a reminder instead.

File: wiki/Skinning_Tutorial/en.md [hunk 9]
```diff
@@ -4,18 +4,18 @@
 
 *For frequently answered skinning questions, see: [Skinning/FAQ](/wiki/Skinning/FAQ)*
 
-## Getting Started
+## Getting started
 
-To get started, open the game client, select [Options](/wiki/Options), and click `Open osu! folder`. Once the `osu!/` folder is opened, go into the `Skins/` folder and create a new folder. You can name it whatever you want, but keep in mind that osu! will display this name in the options. Inside this folder, you can add any skinning elements listed in the skin element lists.
+To get started, open File Explorer. enter the path where you installed osu!, and press enter. Then, go into the `osu!/` folder, then `Skins/`. Right click, and create a new folder. You can also go into osu!, settings, and open the skins folder, then make the folder. You can name it whatever you want, but keep in mind that osu! will display this name on the dropdown of the skin select. Inside this folder, you can add any of the skinning elements listed in the [Skinning Elements Lists](/wiki/Skinning#skin-elements-lists).
```
Comment (sr229) [inline wiki/Skinning_Tutorial/en.md]: Revised version is too complex, revert this.

**Comment by Circleen** [comment]
Kinda unneeded change. Will open a PR for other stuff on this page later.

**Comment by cl8n** [comment]
this is fine to keep open (especially if you are planning to open another PR with similar changes), and if you want to change/shorten your commit history here you can make use of `git rebase` to "squash" commits

usually you don't have to worry about this kind of thing, mergers know what to look out for and how to fix it

**Comment by sr229** [comment]
You shouldn't worry too much on the amount of changes made here since its part of the review process, keep this open please and just edit from then on.

**Pull request closed without merging**